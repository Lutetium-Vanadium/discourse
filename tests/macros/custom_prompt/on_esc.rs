fn main() {
    requestty::questions![Custom { on_esc: todo!() }];
}
