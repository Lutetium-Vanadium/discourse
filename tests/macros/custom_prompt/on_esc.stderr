error: option `on_esc` does not exist for kind `custom`
 --> tests/macros/custom_prompt/on_esc.rs:2:36
  |
2 |     requestty::questions![Custom { on_esc: todo!() }];
  |                                    ^^^^^^
